pkg/kubelet/prober/worker.go

```diff
@@ -132,11 +132,15 @@ func (w *worker) run() {
 	probeTickerPeriod := time.Duration(w.spec.PeriodSeconds) * time.Second
 
 	// If kubelet restarted the probes could be started in rapid succession.
-	// Let the worker wait for a random portion of tickerPeriod before probing.
+	// Let the worker wait for tickerPeriod before probing.
 	// Do it only if the kubelet has started recently.
-	if probeTickerPeriod > time.Since(w.probeManager.start) {
-		time.Sleep(time.Duration(rand.Float64() * float64(probeTickerPeriod)))
```

Comment:
Perhaps we could add a jitter up to the max of the probeTickerPeriod rather than as a multiple of it?

Reply:
Hmm, not sure what we would gain here?

Reply:
Avoiding thundering herd by staggering a small amount. I'll take another look at this later.

Reply:
Ah I see, to wait at least the period plus a various amount... I could also just increment the jitter by one in the manager?

Reply:
FTR I now increment rand.Float64 by 1 to ensure we sleep at least for periodSecond